envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadataValidatorTest.kt

```diff
@@ -221,12 +222,10 @@ class NodeMetadataValidatorTest {
         // expects
         assertThatExceptionOfType(ServiceNameNotProvidedException::class.java)
             .isThrownBy { requireServiceNameValidator.onV3StreamRequest(streamId = 123, request = request) }
-            .satisfies {
-                assertThat(it.status.description).isEqualTo(
-                    "Service name has not been provided."
-                )
+            .satisfies(Consumer {
```

Comment:
Why you used here Consumer explicitly?

Reply:
it's due to this issue: https://github.com/assertj/assertj/issues/2357
adding explicit Consumer seemed like simplest workaround